kerrokantasi/settings/base.py

```diff
@@ -1,12 +1,30 @@
 import os
+
+import environ
+import raven
+
 gettext = lambda s: s
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
-SECRET_KEY = '52k^*)c*bz9t0lzsf_$a+jl3zcy6re!gnw77__)y(#v91-p%tp'
-DEBUG = True
+env = environ.Env()
 
-ALLOWED_HOSTS = []
+DEBUG = env.bool('DEBUG', default=True)
```

Comment:
Is `True` a sane default for `DEBUG`? It sounds like that makes it possible for it to be inadvertently on in production.

Reply:
Good point. I just translated this.